### PR TITLE
Add Exception Manager

### PR DESCRIPTION
**Pull request by NickGrantONS**
# Motivation and Context
To be able to skip messages on Rabbit without any downtime, we need the Exception Manager.

# What has changed
Added Exception Manager to the docker dev config so it starts/stops along with all the other services.

# How to test?
Run `make up`.

# Links
[Trello](https://trello.com/c/f7LmzvJm)

### CODE REVIEW

**Comment by NickGrantONS** [comment]
[SPIKE: zero downtime message quarantine and error log reduction (5 days)](https://trello.com/c/f7LmzvJm/1244-spike-zero-downtime-message-quarantine-and-error-log-reduction-5-days)

**Comment by NickGrantONS** [comment]
[Exception Manager v1.0](https://trello.com/c/2rkYNAMa/1266-exception-manager-v10)

File: rm-services.yml [hunk 1]
```diff
@@ -176,6 +178,22 @@ services:
...
 
+  exceptionmanager:
+    container_name: exceptionmanager
+    image: eu.gcr.io/census-rm-ci/rm/census-rm-exception-manager
+    ports:
+      - "${EXCEPTIONMANAGER_PORT}:8666"
+    external_links:
+     - postgres
+    environment:
+      - FOO=BAR
```
Comment (ryangrundy7) [inline rm-services.yml]: Probably should remove this

File: rm-services.yml [hunk 2]
```diff
@@ -176,6 +178,22 @@ services:
...
+    restart: always
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:8161/actuator/info"]
```
Comment (ryangrundy7) [inline rm-services.yml]: Should be http://localhost:8666/actuator/info

**Pull request merged**